build/tests.dev17.vsconfig

```diff
@@ -101,6 +101,10 @@
     "Microsoft.Component.ClickOnce",
     "Microsoft.VisualStudio.Component.VisualStudioData",
     "Microsoft.Net.Component.4.6.1.SDK",
-    "Microsoft.Net.Component.4.7.2.SDK"
+    "Microsoft.Net.Component.4.7.2.SDK",
+    "Microsoft.Net.Component.4.8.1.SDK",
+    "Microsoft.Net.Component.4.8.1.TargetingPack",
```

Comment:
All the new tests use `ProjectTargetFramework.V48`. Therefore, I don't think that 4.8.1 is needed. Is that right?

I only looked at a single other build to compare to, but this PR's build took 1.5 minutes longer for machine deployment compared to the other build I checked. While that could definitely just be normal variance, installing more things will take longer. Plus execution of the new tests took 11 minutes.

I don't want us to spend a lot of time thinking about test durations. But it does slow down how quickly we can merge pull requests, which multiple people in the team have complained about in multiple end of sprint retrospectives. So I'd like us to spend just a little time to make sure we're not doing something "unreasonable".

Reply:
Hi @zivkan, yes, you are right, we should delete "Microsoft.Net.Component.4.8.1.SDK" & "Microsoft.Net.Component.4.8.1.TargetingPack" from tests.dev17.vsconfig. Thanks for your suggestion.